pullbug/github_bug.py

```diff
@@ -68,7 +68,8 @@ def get_pull_requests(cls, repos, github_owner, github_state):
         for repo in repos:
             try:
                 pull_response = requests.get(
-                    f'https://api.github.com/repos/{github_owner}/{repo["name"]}/pulls?state={github_state}&per_page=100',  # noqa
+                    f'https://api.github.com/repos/{github_owner}/{repo["name"]}/pulls?state={github_state}&per_page=100',
+                    # noqa
```

Comment:
The `# noqa` here should be on the same line as it tells the linter to ignore this line's length which is a tad over the max. Moving it to the next line probably breaks that ignore.